Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /e2e
 
 # Install dependencies
 RUN npm install
-RUN npx playwright install
+RUN npx playwright install chromium
```

Comment:
Please check if this does not influence other e2e tests, use .NET integration-example